### DISCUSSION TITLE
Welcome to govgen-proposals Discussions!

### DISCUSSION THREAD

**Discussion by lbrown2007** [atomone-hub/govgen-proposals#12] [Proposals]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ashrafaltayeb** [comment]
What will be the final supply and how it will be counted and checked before distribution
